auth_user_case_insensitive/i18n/lt.po

```diff
@@ -0,0 +1,37 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* auth_user_case_insensitive
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-04-07 08:58+0000\n"
+"PO-Revision-Date: 2022-04-07 08:58+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: auth_user_case_insensitive
+#: code:addons/auth_user_case_insensitive/hooks.py:0
+#, python-format
+msgid "Conflicting user logins exist for `%s`"
+msgstr "Vartotojams `%s` egzistuoja konfliktuojantys prisijungimai"
```

Comment:
That query checks for uniqueness, no?
```suggestion
msgstr "Vartotojo prisijungimo vardas `%s` jau egzistuoja"
```

Reply:
Sort of, it checks if there are 2 usernames that are identical when lowercased, check happens pre-install

Reply:
Changed it to `Egzistuoja keli vartotojai su prisijungimu `%s``